test/e2e/apps/statefulset.go

```diff
@@ -1154,6 +1154,42 @@ var _ = SIGDescribe("StatefulSet", func() {
 		framework.ExpectNoError(err)
 		e2estatefulset.WaitForStatusAvailableReplicas(c, ss, 1)
 	})
+
+	ginkgo.It("MinReadySeconds should update availableReplicas accordingly when enabled [Feature:StatefulSetMinReadySeconds] [alpha]", func() {
```

Comment:
AvailableReplicas should depend on the MinReadySeconds value specified?

Reply:
updated